src/components/FilterDropdown.jsx

```diff
@@ -23,7 +23,7 @@ function FilterDropdown(props) {
   }
 
   return (
-    <FormControl variant="outlined" className={classes.formControl}>
+    <FormControl variant={variation} className={classes.formControl}>
```

Comment:
I added this to not mess with the styling of the sort filter.

Reply:
I think the sorting filter should actually use the default variant too. Though maybe styled on top of it